Makefile

```diff
@@ -0,0 +1,12 @@
+clipper: target/build/clipper
+
```

Comment:
Please add a copyright header to this file. You can use the `reuse` tool to do this with `reuse annotate --license MPL-2.0 --copyright 'your name' Makefile` or copy paste one of the other headers and put your name in it. 

Sorry about my CI situation, I'll fix it at some point and this will be an automated check.